static/Makefile

```diff
@@ -28,6 +28,15 @@ static-linux: static-cli static-engine ## create tgz with linux x86_64 client an
 		cp -L $(ENGINE_DIR)/bundles/binary-daemon/$$f build/linux/docker/$$f; \
 	done
 	tar -C build/linux -c -z -f build/linux/docker-$(STATIC_VERSION).tgz docker
+	
+	# extra binaries for running rootless
+	mkdir -p build/linux/docker-rootless-extras
+	for f in rootlesskit dockerd-rootless.sh vpnkit; do \
+		if [ -f $(ENGINE_DIR)/bundles/binary-daemon/$$f ]; then \
+			cp -L $(ENGINE_DIR)/bundles/binary-daemon/$$f build/linux/docker-rootless-extras/$$f; \
+		fi \
+	done
+	tar -C build/linux -c -z -f build/linux/docker-rootless-extras-$(STATIC_VERSION).tgz docker-rootless-extras
```

Comment:
As the binaries itself are not specific it would mean duplicating same binaries in two tarballs (and leaving a wrong impression that you need specific binaries to run rootless). Maybe in the future, we can combine them so there is only one tarball, atm. it seems too risky to have the shell scripts and other binaries in the main tarball to me.

> I expect "docker-rootless-extra" to include slirp4netns.

I'm not sure we can do it here because of the GPL licensing (not a lawyer). The launch script should still pick up `slirp4netns` if it is installed through a package manager.